docs/central-users.rst

```diff
@@ -5,92 +5,132 @@ Managing Users in Central
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Comment                          | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View and download                | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Access OData feed                | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
+|                                                                                                      |
+| *Note: Data Collectors can view Entity Lists attached to Forms*                       |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List all                         | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Add properties                   | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update settings                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entities**                                                                                 |
+|                                                                                                      |
+| *Note: Data Collectors create Entities when they fill Entity-creating Forms*                         |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View and download                | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Access OData feed                | x             | x               | x              |                |
```

Comment:
Small thing: how about we list these two actions on the bottom of the section to match submissions? Or update the order for submissions?